src/Raven.Studio/wwwroot/App/views/database/settings/connectionStringElasticSearch.html

```diff
@@ -125,6 +125,25 @@
         </div>
     </div>
 </div>
+<div class="form-group margin-top-lg margin-bottom-sm">
+    <label class="control-label">&nbsp;</label>
+    <div class="text-warning bg-warning padding padding-sm margin-top flex-horizontal">
+        <div class="flex-start"><small><i class="icon-warning"></i></small></div>
+        <div>
+            <small>
+            In order to use the latest <strong>7.x</strong> Elasticsearch client with an <strong>8.x</strong> Elasticsearch server, you must enable the compatibility mode below.<br />
+            This will set <strong>EnableApiVersioningHeader</strong> on the Elasticsearch connection string.
+            </small>
+        </div>
+    </div>
+</div>
+<div class="form-group">
+    <label class="control-label">&nbsp;</label>
+    <div class="toggle">
+        <input id="toggle-mode" type="checkbox" data-bind="checked: enableCompatibilityMode">
```

Comment:
since id needs to be unique on entire page, let's put there something is less collision probablity like: cs-elastic-toggle-mode.